MyCryptoHQ/MyCrypto
# Increase shepherd thresholds

Closes #1723 

### Description

Increases timeout for both providers and queue timeout to 10s. Increases timeout count threshold for providers to default to 10. This is a pretty hard to replicate bug, I feel like the best test would be for @nyeates to try out this commit.

### Steps to Test

1. Use slow 2g/3g connection in devtools
2. Try to send a transaction
3. See if you get a txhash back when its successful
